Explorer/Assets/DCL/PluginSystem/Global/BackpackSubPlugin.cs

```diff
@@ -161,7 +161,7 @@ ICollection<string> forceRender
                     emoteGridController,
                     avatarView.GetComponentsInChildren<AvatarSlotView>().EnsureNotNull(),
                     emotesController,
-                    equippedEmotes
+                    new BackpackCharacterPreviewController(view.CharacterPreviewView, characterPreviewFactory, backpackEventBus, world, equippedEmotes)
```

Comment:
Nitpicking, but I think it would look better if we follow what's done for `emotesController`, as in, declaring a variable outside this `new BackpackController` constructor call.